openmc/tallies.py

```diff
@@ -1653,7 +1659,8 @@ def _align_tally_data(self, other):
             if self_repeat_factor == 1:
```

Comment:
Yeah, I just mean why is there the if/else and not just one option that is always done. For instance, when aligning for nuclide and score, the other tally is always tiled. I'll give this some thought...